recipes/oatpp-swagger/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
+project(oatpp-swagger-test CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(oatpp-swagger REQUIRED)
+
+add_executable(oatpp-swagger-test oatpp-swagger-test.cpp)
```

Comment:
`test_package.cpp` name is preferred if no other reasons

Reply:
Ok, I didn't konw it.